python/helpers/pydev/pydevd.py

```diff
@@ -1358,6 +1358,33 @@ class SetupHolder:
     setup = None
 
 
+def apply_debugger_options(setup_options):
+    """
+
+    :type setup_options: dict[str, bool]
+    """
+
+    default_options = {'save-signatures': False, 'save-threading': False, 'save-asyncio': False, 'qt-support': False}
```

Comment:
That's fine they are always going to be false when connecting from console. You can move them out of this function if you prefer.